contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -229,6 +229,64 @@ END;
 $BODY$
 LANGUAGE plpgsql STABLE;
 
+-- DATABASE_PRINCIPALS: Include Hard coded public, sys, INFORMATION_SCHEMA users
```

Comment:
I think we usually rename existing view to another name and create a new one in the upgrade script

Reply:
added 

```
ALTER VIEW sys.database_principals RENAME TO database_principals_deprecated_3_2_0;
CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'database_principals_deprecated_3_2_0');
```